.github/workflows/test.yml

```diff
@@ -34,3 +34,65 @@ jobs:
       - run: yarn test
         env:
           CI: true
+
+  build-ios:
```

Comment:
we already have build-ios, and similar job names can confuse GH, make this e2e-ios